test/runtests.jl

```diff
@@ -1315,6 +1339,27 @@ end
                 result = execute(conn, "SELECT '(2018-01-01, 2018-02-02]' = \$1", [Interval(Date(2018, 1, 1), Date(2018, 2, 2), false, true)])
                 @test first(first(result))
                 close(result)
+
+            end
+
+            @testset "InfExtendedTime" begin
+                tests = (
+                    ("'infinity'::date", InfExtendedTime{Date}(∞)),
+                    ("'-infinity'::date", InfExtendedTime{Date}(-∞)),
+                    ("'infinity'::timestamp", InfExtendedTime{DateTime}(∞)),
+                    ("'-infinity'::timestamp", InfExtendedTime{DateTime}(-∞)),
+                    ("'infinity'::timestamptz", InfExtendedTime{ZonedDateTime}(∞)),
+                    ("'-infinity'::timestamptz", InfExtendedTime{ZonedDateTime}(-∞)),
+                    ("'(-infinity, 2012-01-01]'::daterange", Interval{Open, Closed}(-∞, Date(2012, 1, 1))),
```

Comment:
Do tests like this with DateTime and ZonedDateTime too (protects against `Infinity.jl` changing printing for those types)